devito/finite_difference.py

```diff
@@ -137,18 +137,23 @@ def first_derivative(*args, **kwargs):
     dim = kwargs.get('dim', x)
     diff = kwargs.get('diff', h)
     order = kwargs.get('order', 1)
-    side = kwargs.get('side', 1)
+    side = kwargs.get('side', 0)
```

Comment:
I guess it's fine, I liked it like that because it was defining the sign in front of the output and avoid str comparison.